draft-mishra-scone-usecase.md

```diff
@@ -272,56 +284,59 @@ SCONE signaling can be integrated into existing operational and
 management frameworks to enable monitoring, troubleshooting, and fault
 isolation. Metrics of interest include:
 
-- Rate of SCONE advisory messages issued per session  
-- Correlation between SCONE advisories and user-plane throughput changes  
-- Error conditions where SCONE signaling fails to reach the intended
-endpoints
+- Rate of SCONE advisory messages issued per session
+- Correlation between SCONE advisories and user-plane throughput changes
+- Error conditions where SCONE signaling fails to reach the intended endpoints
 
 ## Conformance Monitoring
 Networks providing SCONE throughput advice ought to implement
 mechanisms to measure compliance, either per application flow or in
 aggregate. This allows operators to validate advisory effectiveness and
-adjust policies. Due to flow awareness, such mechanisms are typically 
-implemented in a SCONE Network Element but may also be implemented 
+adjust policies. Due to flow awareness, such mechanisms are typically
+implemented in a SCONE Network Element but may also be implemented
 elsewhere in the network.
 
 ## Standards Compliance
-SCONE signaling is expected to traverse the existing data path. For
-example, in 3GPP-compliant networks, SCONE packets are carried within
-Protocol Data Unit (PDU) sessions established between the User Equipment
-(UE) and Internet endpoints.
+SCONE signaling is expected to traverse the existing data path associated
+with the IP flow for which the Network Element intends to send the advisory bit-rate.
+
+For example, in 4G mobile networks SCONE packets are sent by the Network Element
+(i.e., the P-GW) over the corresponding EPS bearer session (S5U Interface + S1-U Interface + Air Interface)
+that is carrying the IP flow which the Network Element intends to send the advisory bit-rate.
+
+Similarly, in 5G mobile SCONE packets are sent by the Network Element (i.e., the UPF) over the
+corresponding PDU session (N3 Interface + Air Interface) that is carrying the IP flow for which
+the Network element intends to send the bit-rate.
```

Comment:
Why are we introducing so much 3GPP details here again? I would simply remove the examples, if we want to change anything.